modules/claims_api/app/models/claims_api/power_of_attorney.rb

```diff
@@ -13,17 +13,18 @@ class PowerOfAttorney < ApplicationRecord
 
     PENDING = 'pending'
     UPDATED = 'updated'
+    SUBMITTED = 'submitted'
     ERRORED = 'errored'
 
-    before_validation :set_md5
-    validates :md5, uniqueness: true
+    before_save :set_md5
```

Comment:
why change this to before_save?

Reply:
since this removes the md5 uniqueness validation, I don't think there's any validations that explicitly get run (unless there are implicit ones that still do?), so I figured just explicitly doing it pre-save works just as well